### ISSUE TITLE
Update VAMC Health Services template with components (v3)

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16997]
## Description
VAMC Health Service pages need the following updates to components:

Example pages:

- [VAMC Boston Health Services](https://www.va.gov/boston-health-care/health-services/)
- [VAMC Minneapolis health Services](https://www.va.gov/minneapolis-health-care/health-services/)

### V1 components that need to be upgraded (Actionable)
1. Accordion
   - Accordion sections (Prepare for your visit and services) are using the v1 component and need to be updated [to v3](https://design.va.gov/components/accordion#examples---v3). 

### Not in scope
* #17657

## Acceptance Criteria
- [x] Accordion component has been updated to v3
- [x] Update and run existing test(s).
- [x] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Presentation is verified the same across all supported browsers on Mac & Win
    - [ ] Chrome 
    - [ ] Firefox
    - [ ] Safari
    - [ ] Edge
    - [ ] Android
    - [ ] iOS
- [ ] Requires accessibility review

**Comment by eselkin** [comment]
Do we have a ticket for the "In the Spotlight" boxes (which will eventually move to Cards) to change the links to `<va-link active>` links? There are slight font-weight changes since `<va-link>` in the active state are larger weight, but they improve in the highlighting and focus and remove icons.

**Comment by jilladams** [comment]
Due to #17807 popping up, where adoption of components ran into some browser-specific styling issues, I've added ACs here to cover cross-browser verification.

**Comment by jilladams** [comment]
@eselkin No: we haven't ticketed the build updates to in the spotlight. I added your note to a comment on the epic where i"m tracking the next ticketing work we need to get done: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15931#issuecomment-2021836133

And just checking: we wouldn't make the `va-active-link` update now because it's out of line with the rest of the current spotlight treatment?

**Comment by laflannery** [comment]
@jilladams and @eselkin We are able to move forward on updating the spotlights to use the card component, it's ticketed in the ["not in scope" ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17657) from the description above along with the other imposter/non-component work which we are currently de-prioritizing. The work Jill noted isn't blocking the FE work to update to the card component.

Also, regarding the link update Eli mentioned, I purposefully didn't have this in either ticket (this or the other) because the work on spotlights will be updating to the Action link VADS class.

Let me know if that helps answer questions/clarify

**Comment by laflannery** [comment]
## Screenshots from Eli's PR for ACs:

**FF Windows**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/907895f9-ac33-4f57-b0ad-fc5fc53c2052)

**Edge** (I didn't put the fancy bookmarklet on Edge but confirmed these are the v3 variant)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a7f62cd9-9ddd-43cd-a288-f83306964c8e)

**Chrome Windows**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/2518ac3a-190c-4703-aadd-8b5c03e05b42)

_Will do tomorrow morning_
**FF Mac**

**Chrome Mac**

**Safari**

Android and iOS can't be tested until Staging/Prod unless we use ngrok..pending answer from other ticket on how to handle

**Comment by jilladams** [comment]
After discussing with Laura: 
* Because phones can't access SOCKS, we can't verify on Tugboat or Review Instances using actual phones. 
* We can verify using: 
    * ngrok
    * desktop browser set to a mobile / tablet breakpoint, using something like [user-agent switcher extension](https://chromewebstore.google.com/detail/user-agent-switcher-for-c/djflhoibgkdhkhhcedjiklpkjnoahfmg?pli=1)

We don't think it's the best use of our engineering time to stand up ngrok for all of these, so if we've verified using emulator / dev tools, that's ok for merge and we can prod verify on phones. 

@laflannery: lemme know if I misstated any of that discussion. 
@eselkin  (and @maxx1128 ) lemme know if you have any questions about that. 

cc @mmiddaugh for awareness
